test/python/primitives/test_backend_sampler.py

```diff
@@ -358,6 +374,28 @@ def test_sequential_run(self):
         self.assertDictAlmostEqual(result3.quasi_dists[0], {0: 1}, 0.1)
         self.assertDictAlmostEqual(result3.quasi_dists[1], {1: 1}, 0.1)
 
+    def test_bound_pass_manager(self):
+        """Test bound pass manager."""
+
+        bound_counter = LogPass("bound_pass_manager")
+        bound_pass = PassManager(bound_counter)
+
+        sampler = BackendSampler(
+            backend=FakeNairobi(), skip_transpilation=True, bound_pass_manager=bound_pass
```

Comment:
To pass the test, how about removing `skip_transpilation=True`?

Reply:
Yes, I had kept it because it passed locally in my environment. Thanks